docs/reference/search/search.asciidoc

```diff
@@ -164,7 +207,28 @@ Period to retain the <<scroll-search-context,search context>> for scrolling. See
 By default, this value cannot exceed `1d` (24 hours). You can change
 this limit using the `search.max_keep_alive` cluster-level setting.
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=search_type]
+[[search-type]]
+`search_type`::
+(Optional, string)
+How {wikipedia}/Tf–idf[distributed term frequencies] are calculated for
```

Comment:
This is a nice simplification. I think the original page was designed when we also supported a `query_and_fetch` parameter, and so included extra background.